### ISSUE TITLE
Date of ADRs shown in Structurizr Lite Decisions is one day earlier than in Markdown file

### ISSUE THREAD

**Issue by prlcutting** [structurizr/ui#23]
### Description

When creating an ADR with a specific date in the Markdown file, I expect the same date to show up when rendered in Structurizr Lite's Decisions views. However, the date shown is one day earlier.

### Steps to reproduce

1. Create an ADR in Markdown format with a specific date using ADR Tools (or manually).
2. View the ADR in Structurizr Lite.
3. Note that the date listed in Structurizr Lite is one day earlier than that listed in the ADR Markdown file.

### Screenshot

# ADR Markdown files

![image](https://github.com/structurizr/ui/assets/6989884/058dbb16-380f-4a68-8919-d80f289bc9cc)

![image](https://github.com/structurizr/ui/assets/6989884/05d31e23-7994-4f08-9350-093d11fa3e67)

# Structurizr Lite Decisions view

![image](https://github.com/structurizr/ui/assets/6989884/68b9156c-63ee-417c-9dd6-663d27bc0810)

> Please excuse the odd numeric sequencing. I'm creating these ADRs retroactively, and not necessarily in chronological order.

### Severity

Minor

### Priority

Low

### Resolution

I have no budget, please fix this for free

**Comment by simonbrowndotje** [comment]
This is likely a time zone issue ... either when parsing the ADRs, or when rendering them. You could try setting `structurizr.timezone` (see https://structurizr.com/help/properties). If that doesn't resolve the issue, then an example ADR file and the resulting `workspace.json` file (so I can see the dates in both) would be helpful. Thanks!

**Comment by prlcutting** [comment]
Hi @simonbrowndotje - could you please advise or point me to documentation describing where I would actually set `structurizr.timezone`? I followed the link to the Properties documentation that you provided, but I'm not sure where I'm supposed to specify this. Thanks.

**Comment by prlcutting** [comment]
I created a `structurizr.properties` file adjacent to my `workspace.dsl` and `workspace.json` files with the following content:

```text
structurizr.timezone=America/New_York
```

I then refreshed the Decisions page in Structurizr Lite, but didn't observe any change in the dates that were displayed. I'm not sure if that's because the file is in the wrong place, I've configured things wrong, e.g. bad format in the file, or this just isn't the fix for the problem.

I stumbled across [this](https://docs.structurizr.com/lite/usage) documentation too. It talks about setting values in a `structurizr.properties` file, but doesn't state where that file should go. It also describes different values than are listed in the documentation [here](https://structurizr.com/help/properties), so not sure if they're related or not.

Any guidance would be appreciated, thanks.

**Comment by simonbrowndotje** [comment]
`structurizr.timezone` is a viewset property, so needs to be specified on the viewset; for example:

```
workspace {

    model {
        user = person "User" 
        softwareSystem = softwareSystem "Software System"

        user -> softwareSystem "Uses"
    }

    views {
        systemContext softwareSystem "SystemContext" {
            include *
            autoLayout
        }

        properties {
            "structurizr.timezone" "Australia/Sydney"
        }
    }
    
}
```

**Comment by prlcutting** [comment]
Thanks for the info @simonbrowndotje. That helped me set the time zone properly. Here's what I learned:

If the time zone is implicitly (i.e., presumably read from system settings), or explicitly (i.e., configured with `structurizr.timezone`) UTC or later, then dates in the Decisions page are correct. However, if the time zone is explicitly set to anything other than "my" time zone, timestamps in the Diagrams page are now "wrong"; that is, they're labelled correctly, just not in my time zone, which is unnatural.

Since my time zone is `America/New_York`, i.e., before UTC, then the ADR dates are wrong. The same would be true for any time zone before UTC. I tested a couple to prove this.

So it seems like the date in the ADR is assumed to be in UTC, and if you happen to be in an earlier time zone (either implicitly, or explicitly), then the dates in the Decisions page will appear to be one day earlier.

Hope this makes sense.

**Comment by simonbrowndotje** [comment]
Yes, UTC is used by default (https://github.com/structurizr/import/blob/main/src/main/java/com/structurizr/importer/documentation/AdrToolsDecisionImporter.java#L48) but there's no way to easily change that if you're using the `!adrs` keyword from the DSL, so I'm movingthis issue to the DSL repo.
